telos-frontend/src/views/journal/Page.jsx

```diff
@@ -1,24 +1,48 @@
-import { useCallback } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { useDrop } from 'react-dnd';
-import update from 'immutability-helper';
 import DraggableWidget from '../../dnd/DraggableWidget';
 import { WidgetTypes } from '../../dnd/WidgetTypes';
 import pageStyles from './Page.module.css';
 
-// eslint has been disabled here as the unused args date and leftPage might be used in future.
-// eslint-disable-next-line no-unused-vars
+const axios = require('axios');
+
 const Page = ({ date, widgets, setWidgets }) => {
+  // Initial widget data from the backend. Passed down to the widgets as their initial state.
+  const [initialWidgetData, setInitialWidgetData] = useState({});
+
+  // Fetch widgets for page/date
+  useEffect(() => {
+    axios.get(`http://localhost:3001/api/journal/${date}`).then(({ data }) => {
```

Comment:
Same as prev comment. Might be worth making a deferred binding change to all axios.get() calls